### PR TITLE
sample to scale app service plan

### PR DESCRIPTION
**Pull request by yungezz**
## Purpose

* ...
sample to scale app service plan.

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

## How to Test
*  Install ansible

```bash
$ pip install ansible[azure]
```

*  [Optional] Install ansible role

```bash
$ ansible-galaxy install Azure.azure_preview_modules
$ pip install -r ~/.ansible/roles/Azure.azure_preview_modules/files/requirements-azure.txt
```

*  Get the sample playbook

```
git clone https://github.com/Azure-Samples/ansible-playbooks.git
cd ansible-playbooks
git checkout [branch-name]
```

* Test the code

```bash
ansible-playbook <filename>.yml
```

## What to Check
Verify that the playbook is successfully run.

### CODE REVIEW

**Comment by Fred-sun** [comment]
@zikalino Please help to review this PR. Thanks!

File: appserviceplan_scale.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#
+# This playbook requires Ansible version >=2.7, or you could use role Azure.azure_preview_modules
+# If you're using Ansible >=2.7, just remove the role lines.
+
+- hosts: localhost
+  connection: local
+  vars:
+    location: eastus
+    plan_name: myAppServicePlan
+
```
[the remark this answers is not in the document]
Comment (yungezz) [reply appserviceplan_scale.yml]: it's a global variable defined as ansible-playbook varabible

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino Please help to review this PR. Thanks!

**Comment by Fred-sun** [comment]
@zikalino

**Comment by Fred-sun** [comment]
@zikalino @yuwzho Please help review this PR and push for merged! Thanks!

**Pull request merged**